docs/tutorials/basic/symbol.md

```diff
@@ -349,7 +296,8 @@ print('number of outputs = %d\nthe first output = \n%s' % (
 
 We can evaluate the same symbol on GPU with different data.
 
-**Note** In order to execute the following section on a cpu set gpu_device to mx.cpu().
+**Note** In order to execute the following section on a cpu set `gpu_device` to `mx.cpu()`.
```

Comment:
Make automatic. I don't know how this tutorial passes in CI? Aren't all tutorials tested on CPU too?